components/Posts/PostCard/Tags/index.jsx

```diff
@@ -13,4 +14,12 @@ const Tags = (props) => {
     }
 }
 
+Tags.propTypes = {
+    props: PropTypes.object,
+};
+
+Tags.defaultProps = {
+    props: {},
```

Comment:
No need to add default props.
If you write the default props then we need to check the properities, better if you write propTypes required without defaults.